e2e/tests/user/registration.spec.ts

```diff
@@ -0,0 +1,122 @@
+    expect(createAccountButton).toBeTruthy();
+    await createAccountButton.click();
+
+    // Step 1: Enter invitation key.
+    const createAccountPage = new CreateAccountPage(page);
+    const header = 'Enter your Invitation Key:';
+    const headerDisplayed = await waitForText(page, 'h2', header);
+    expect(headerDisplayed).toBeTruthy();
+
+    await createAccountPage.fillOutInvitationKey();
+    await page.waitFor(1000);
+
+    // Step 2: Accepting Terms of Service.
+    const pdfPageCount =  await page.waitForFunction(() => {
+      return document.querySelectorAll('.tos-pdf-page[data-page-number]').length === 9
+    }, {timeout: 30000});
+    // expecting 9 pages in pdf document
+    expect(await pdfPageCount.jsonValue()).toBe(true);
+
+    await createAccountPage.acceptTermsOfUseAgreement();
+    let nextButton = await createAccountPage.getNextButton();
+    await nextButton.click();
+    await page.waitFor(1000);
+
+    // Step 3: Enter user information
+    // Should be on Create your account: Step 1 of 2 page
+    expect(await waitUntilFindTexts(page, 'Create your account')).toBeTruthy();
+    await createAccountPage.fillOutUserInformation();
+    nextButton = await createAccountPage.getNextButton();
+    await nextButton.click();
+    await page.waitFor(1000);
+
+    // Step 4: Enter demographic survey (All Survey Fields are optional)
+    // Should be on Demongraphic Survey page
+    const demograpicsPageHeader = 'Demographics Survey (All Survey Fields are optional)';
+    expect(await waitForText(page, 'h3', demograpicsPageHeader)).toBeTruthy();
+    await createAccountPage.fillOutDemographicSurvey();
+    const submitButton = await createAccountPage.getSubmitButton();
+    await submitButton.click();
+    await page.waitFor(1000);
+
+    // Step 5: New account created successfully page.
+    const congratHeader = 'Congratulations!';
+    await page.waitForSelector('h1', { visible: true });
+
+    const h4List = [];
+    const h4Headers: ElementHandle[] = await page.$$('h4');
+    for (const h4Header of h4Headers) {
+      const txt = await (await h4Header.getProperty('innerText')).jsonValue();
+      h4List.push(txt);
+    }
+    const expectedH4Headers = [
+      'Your All of Us research account has been created!',
+      'Check your contact email for instructions on getting started.',
+      `Your contact email is: ${process.env.CONTACT_EMAIL}`
+    ];
+    expect(h4List).toEqual(expectedH4Headers);
+
+    const resendButton = await page.waitForXPath('//button[.="Resend Instructions"]');
+    expect(resendButton).toBeTruthy();
+
+    const changeEmailButton = await page.waitForXPath('//button[.="Change contact email"]');
+    expect(changeEmailButton).toBeTruthy();
+
+    const h3List = [];
+    const h3Headers: ElementHandle[] = await page.$$('h3');
+    for (const h3Header of h3Headers) {
+      const txt = await (await h3Header.getProperty('innerText')).jsonValue();
+      h3List.push(txt);
+    }
+    console.log(h3List);
```

Comment:
print out new account message.
![Screen Shot 2020-02-20 at 5 13 00 PM](https://user-images.githubusercontent.com/35533885/74984928-7fdcca80-5404-11ea-9b26-8ff8126390b7.png)

Reply:
prefix `aoutestuser` is used to allow easy finding to delete in gsuite in future.